src/User/Command/EditUserHandler.php

```diff
@@ -78,17 +77,18 @@ public function handle(EditUser $command)
                     $validate['email'] = $attributes['email'];
                 }
             } else {
-                $actor->assertPermission($canEdit);
+                $actor->assertPermission($actor->can('editCredentials', $user));
                 $user->changeEmail($attributes['email']);
             }
         }
 
-        if ($actor->isAdmin() && ! empty($attributes['isEmailConfirmed'])) {
+        if (! empty($attributes['isEmailConfirmed'])) {
+            $actor->assertAdmin();
             $user->activate();
         }
 
         if (isset($attributes['password'])) {
-            $actor->assertPermission($canEdit);
+            $actor->assertPermission($actor->can('editCredentials', $user));
             $user->changePassword($attributes['password']);
```

Comment:
off topic observation: we force users to use the change password button (reset password email) on the frontend but not on the backend, so they could just send a payload and directly edit it.

I don't know if this was done on purpose.

Reply:
I don't disagree, but this PR already contains a lot so I'd leave this alone for now

Reply:
Yea, I wouldn't want that changed here anyway :P was an observation as it looked odd.